lib/frontend/post-ghae-complete.js

```diff
@@ -22,38 +22,44 @@ module.exports = function (opts) {
 
     // check valid referer, needed to validate ghaehost
     if (!req.get('referer')) {
-      handleExceptions(res, 400, 'Request not coming from valid referer');
-      return;
+      return res.status(400).render('ghae-register-error.hbs', {
+        error: 'Request not coming from valid referer',
```

Comment:
can we add trace here, we need to trace the source request IP. For now we can use the existing logger base utility